app/components/agenda/agendaitem/agendaitem-decision.hbs

```diff
@@ -96,20 +96,44 @@
         class="au-o-flow {{if this.isEditing 'au-u-background-gray-100'}}"
       >
         <h4 class="auk-h4">{{t "concerns"}}</h4>
-        <Agenda::Agendaitem::Decision::Editor
+        <SayEditor
+          @allowFullscreen={{true}}
           @handleRdfaEditorInit={{this.handleRdfaEditorInitBetreft}}
-          @piecePart={{this.betreftPiecePart}}
-          @onPiecePartVersionClicked={{this.setBetreftEditorContent}}
-          @onUpdateContentClicked={{this.updateBetreftContent}}
-        />
+        >
+          <:header>
+            <AuButton
+              @skin="link"
+              @icon="renew"
+              class="au-u-padding-left-none"
+              {{on "click" this.updateBetreftContent}}
+            >{{t "update-thing" thing=(t "content")}}</AuButton>
+            <VersionsDropdown
+              @record={{this.betreftPiecePart}}
+              @previousVersionKey="previousPiecePart"
+              @onVersionClicked={{this.onRevertBetreftVersion}}
+            />
+          </:header>
+        </SayEditor>
         <h4 class="auk-h4 au-u-padding-top">{{t "agendaitem-decision"}}</h4>
-        <Agenda::Agendaitem::Decision::Editor
```

Comment:
This component is no longer used and can be removed?
`<Agenda::Agendaitem::Decision::Editor>`

Reply:
`Agenda::Agendaitem::Decision::PiecePartVersion` is also no longer used if the above is removed.

Reply:
Yes you're absolutely right, I forgot to remove them

Reply:
fixed in [ec2687f](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1784/commits/ec2687f5232923469afa09cef7f1d59c2dbe9746)